core/src/main/kotlin/org/evomaster/core/output/naming/ActionTestCaseNamingStrategy.kt

```diff
@@ -0,0 +1,27 @@
+package org.evomaster.core.output.naming
+
+import org.evomaster.core.problem.httpws.HttpWsCallResult
+import org.evomaster.core.problem.rest.RestCallAction
+import org.evomaster.core.search.EvaluatedIndividual
+import org.evomaster.core.search.Solution
+
+open class ActionTestCaseNamingStrategy(
+    solution: Solution<*>
+) : NumberedTestCaseNamingStrategy(solution)  {
+
+
+    override fun expandName(individual: EvaluatedIndividual<*>): String {
+        var evaluatedAction = individual.evaluatedMainActions().last()
+        var action = evaluatedAction.action as RestCallAction
+        var result = evaluatedAction.result as HttpWsCallResult
+        return "_${action.verb}_on_${sanitizePathName(action.path.nameQualifier)}_returns_${result.getStatusCode()}"
+    }
+
+    private fun sanitizePathName(nameQualifier: String): String {
```

Comment:
this type of function should be in a utlity class, like for example `TestWriterUtils`.
Also there is already something that could be used, eg, `safeVariableName()`